news/5400.bugfix

```diff
@@ -0,0 +1 @@
+Solved issue #5400 by adding label for Search Block Dropdown Form element. @pranayjoshi
```

Comment:
By virtue of this filename, the change log will automatically include the issue number when it is released.

```suggestion
Improved accessibility by adding a label for the search block dropdown form element. @pranayjoshi
```

Reply:
Thanks. Would mind adding the review badge to my other PR.